docs/docs/examples/backend.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Backend / API
+---
+
+### Scope
+
+To make our examples more accessible and user-friendly, we've outlined the capabilities that each example includes.
+By consulting the capabilities list, you can quickly determine whether a particular example has the features you need.
+
+As a general rule, we only include examples that meet the capabilities listed on our site.
+If we do have an SDK available for a particular example, we'll use it.
+Otherwise, we rely on existing authentication libraries.
+
+**Authorization**
+
+In the examples we will use a service account with a personal access token which can be used directly to access the example API.
+
+**RBAC**
+
+To show how role-based access control (RBAC) works the examples include multiple endpoints.
+- Public Endpoint: An endpoint that is accessible by everyone, no matter if the user is authenticated or has some roles.
+- Private Endpoint: And endpoint where the is has to be authenticated (Authentication Header) to be able to call it.
+- Private Endpoint with Role: The user has to be authenticated and have a specific role/permission "read:messages" to be able to call the endpoint
+
+**Check token against introspection_endpoint**
+The token of a requesting user will be checked against the [introspection_endpoint](../apis/openidoauth/endpoints#introspection_endpoint)
+For this the roles scopes will be used instead of the "assert roles on authentication" setting, that can be configured on the project.
+
+### Examples
+
+<table>
+  <tr>
+    <th></th>
+    <th>Language</th>
+    <th>Authorization</th>
+    <th>RBAC</th>
+    <th>Introspection</th>
```

Comment:
I think we should not show this information, but should use it as internal guideline.